packages/ingest-mongodb-public/src/sources/snooty/SnootyDataSource.test.ts

```diff
@@ -208,6 +209,17 @@ describe("SnootyDataSource", () => {
       expect(pages).toHaveLength(1);
       noIndexMock.done();
     });
+
+    it("includes tocIndex in metadata", async () => {
+      const source = await makeSnootyDataSource({
+        name: `snooty-test`,
+        project,
+        snootyDataApiBaseUrl,
+      });
+      const pages = await source.fetchPages();
+
+      expect(pages[0].metadata?.page?.tocIndex).toBe(0);
```

Comment:
Not too familiar with snooty toc internals - do we just have a number or can we get some kind of TOC breadcrumb string?

e.g. something like `breadcrumb: "Indexes > Create"` is more useful at a glance than `tocIndex: 112`

Reply:
just have a number. and this is just to be used for things like concating page summaries a la skunkworks. doesn't get into embedded content. so just having index is sufficient for that